src/main/java/in/zapr/druid/druidry/dimension/ListFilteredDimension.java

```diff
@@ -0,0 +1,29 @@
+package in.zapr.druid.druidry.dimension;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import in.zapr.druid.druidry.dimension.enums.FilteredDimensionType;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+
+import java.util.List;
+
+@Getter
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@EqualsAndHashCode(callSuper = true)
+public class ListFilteredDimension extends FilteredDimension {
```

Comment:
There may arise use case in future where some type of new filter would extend These filters. Because of this I chose it to be protected. It makes sense to mark them as private otherwise.

Reply:
I agree with @abhi-zapr.
We can always give more access but once given, removal is painful.

Reply:
acknowledged.